build.cmd

```diff
@@ -28,27 +32,48 @@ if '!_FloatingDependencies!'=='0' (
   set _defaultBuildConfig=/p:Configuration=Windows_NT_Debug
   )
 )
+:: *** end WCF Content ***
+
+:ReadArguments
+:: Read in the args to determine whether to run the native build, managed build, or both (default)
+set "__args=%*"
+if /i [%1] == [native] (set __buildSpec=native&&set "__args=%__args:~6%"&&shift&&goto Tools)
```

Comment:
We don't have any native libraries to build, right? Won't this cause the script to bail?